testcases/kernel/syscalls/sigrelse/sigrelse01.c

```diff
@@ -733,6 +401,7 @@ int choose_sig(int sig)
 	case SIGALRM:
 	case SIGCANCEL:
 	case SIGTIMER:
+	case SIGSYNCCALL:
```

Comment:
Signals 32, 33, and 34 are used by musl.  We should skip any signals in the `SIGRTMIN` to `SIGRTMAX` range.  These are reserved for userspace and may be used by libc.

Reply:
Modified and submitted a new commit.